addons/nametags/initSettings.sqf

```diff
@@ -0,0 +1,56 @@
+// CBA Settings [ADDON: ace_nametags]:
+
+[
+    QGVAR(defaultNametagColor), "COLOR",
+    [LSTRING(DefaultNametagColor)],
+    [format ["ACE %1", localize LSTRING(Module_DisplayName)], localize "str_a3_rscdisplaygameoptions_buttongui"],
+    [0.77, 0.51, 0.08, 1],
+    false, // isGlobal
+    {[QGVAR(defaultNametagColor), _this] call EFUNC(common,cbaSettings_settingChanged)}
+] call CBA_settings_fnc_init;
+
+[
+    QGVAR(nametagColorMain), "COLOR",
+    ["str_team_main"],
+    [format ["ACE %1", localize LSTRING(Module_DisplayName)], localize "str_a3_rscdisplaygameoptions_buttongui"],
+    [1.00, 1.00, 1.00, 1],
+    false, // isGlobal
+    {[QGVAR(nametagColorMain), _this] call EFUNC(common,cbaSettings_settingChanged)}
+] call CBA_settings_fnc_init;
+
+[
+    QGVAR(nametagColorRed), "COLOR",
+    ["str_team_red"],
```

Comment:
Maybe `[format ["%1 %2",localize "str_team", localize "str_team_red"]],`
so we get "Team Red"?

Reply:
I was considering this but I'm not proficient enough in all the languages to know if the word order for that would be grammatically correct (for Finnish it would be reversed...)